weaviate/collection/classes/data.py

```diff
@@ -0,0 +1,89 @@
+import uuid as uuid_package
+from dataclasses import dataclass
+from typing import Any, List, Optional, Dict, Union
+
+from pydantic import BaseModel, Field
+from weaviate.util import _to_beacons
+from weaviate.weaviate_types import UUID
+
+
+class IncludesModel(BaseModel):
+    def to_include(self) -> str:
+        include: List[str] = []
+        for field, value in self:
+            if value:
+                include.append(field)
+        return ",".join(include)
+
+
+class GetObjectByIdIncludes(IncludesModel):
```

Comment:
Not sure when, but looking at it again now I think `metadata` here would be confusing since that is also used when accessing the `_Object` class and the `metadata` here is not related to the `metadata` there. Or is `metadata` suitably general so as not to be confusing between contexts?

Reply:
hmm it both specifies which parts of the obejct metadata should be returned and they are both accessible under object.metadata.X. The problem is that the REST-API sends some by default (which we always include) and therefore we cannot use the same Metadata return object for both. "Includes" sounds like the REST-API below, which we want to hide from the user

`GetObjectByIdMetadata`?